### PR TITLE
ADBDEV-6047: Add OpenJDK 11 to Ubuntu image

### PR DESCRIPTION
**Pull request by RekGRpth**
Add OpenJDK 11 to Ubuntu image

We need Java for PXF, ADBC etc, so this patch adds OpenJDK 11 to Ubuntu image,
as it did for CentOS 7 image.

### CODE REVIEW

**Comment by BenderArenadata** [comment]
Allure report https://allure.adsw.io/launch/79396

**Review by silent-observer — DISMISSED** [review]

File: arenadata/Dockerfile.ubuntu [hunk 1]
```diff
@@ -24,10 +24,13 @@ RUN set -eux; \
 # Alter precedence in favor of IPv4 during resolving
     echo 'precedence ::ffff:0:0/96  100' >> /etc/gai.conf; \
 # Packages for tests
-    DEBIAN_FRONTEND=noninteractive apt install -y krb5-kdc krb5-admin-server fakeroot sudo python-pip; \
+    DEBIAN_FRONTEND=noninteractive apt install -y krb5-kdc krb5-admin-server fakeroot sudo python-pip openjdk-11-jdk; \
 # Install allure-behave for behave tests
     pip2 install allure-behave==2.4.0; \
-    pip2 cache purge
+    pip2 cache purge; \
+    echo '#!/bin/sh' >> /etc/profile.d/jdk_home.sh; \
```
Comment (andr-sokolov) [inline arenadata/Dockerfile.ubuntu]: Can we use one command to create `/etc/profile.d/jdk_home.sh` instead of three?
Can we replace `>>` with `>`?

Comment (RekGRpth) [reply arenadata/Dockerfile.ubuntu]: I only could combine it into one line

Comment (RekGRpth) [reply arenadata/Dockerfile.ubuntu]: reverted and use one `>` at first place

**Review by andr-sokolov — APPROVED** [review]

**Comment by silent-observer** [comment]
Was JAVA_HOME not necessary in the end? Or will this still be left in PXF dockerfile?

**Comment by andr-sokolov** [comment]
> Was JAVA_HOME not necessary in the end? Or will this still be left in PXF dockerfile?

@silent-observer , can you explain, what happens without JAVA_HOME?

**Comment by RekGRpth** [comment]
> Was JAVA_HOME not necessary in the end? Or will this still be left in PXF dockerfile?

We decided move it to PXF Dockerfile

**Comment by silent-observer** [comment]
> > Was JAVA_HOME not necessary in the end? Or will this still be left in PXF dockerfile?
> 
> @silent-observer , can you explain, what happens without JAVA_HOME?

@andr-sokolov, checked, seems like Gradle can still find Java without an issue. Maybe it's not necessary even in PXF?

**Review by silent-observer — APPROVED** [review]

**Comment by RekGRpth** [comment]
> > > Was JAVA_HOME not necessary in the end? Or will this still be left in PXF dockerfile?
> > 
> > 
> > @silent-observer , can you explain, what happens without JAVA_HOME?
> 
> @andr-sokolov, checked, seems like Gradle can still find Java without an issue. Maybe it's not necessary even in PXF?

It can't [find](https://github.com/arenadata/pxf/blob/ef2d606456936f597e51f7f3daff4ae9d9bc4f9c/server/gradlew#L86-L102) Java without JAVA_HOME env var

**Comment by BenderArenadata** [comment]
Allure report https://allure.adsw.io/launch/79440

**Comment by BenderArenadata** [comment]
Failed job Regression tests with ORCA on x86_64: https://gitlab.adsw.io/arenadata/github_mirroring/gpdb/-/jobs/1866607

**Comment by BenderArenadata** [comment]
Failed job Regression tests with Postgres on x86_64: https://gitlab.adsw.io/arenadata/github_mirroring/gpdb/-/jobs/1866606

**Comment by BenderArenadata** [comment]
Allure report https://allure.adsw.io/launch/79468

**Comment by BenderArenadata** [comment]
Failed job Regression tests with ORCA on x86_64: https://gitlab.adsw.io/arenadata/github_mirroring/gpdb/-/jobs/1867731

**Comment by BenderArenadata** [comment]
Failed job Regression tests with Postgres on x86_64: https://gitlab.adsw.io/arenadata/github_mirroring/gpdb/-/jobs/1867730

**Comment by BenderArenadata** [comment]
Failed job Regression tests with ORCA on x86_64: https://gitlab.adsw.io/arenadata/github_mirroring/gpdb/-/jobs/1868727

**Comment by BenderArenadata** [comment]
Failed job Regression tests with Postgres on x86_64: https://gitlab.adsw.io/arenadata/github_mirroring/gpdb/-/jobs/1868728

**Comment by RekGRpth** [comment]
bender build

**Comment by BenderArenadata** [comment]
Failed job Deploy multiarch Dockerimages: https://gitlab.adsw.io/arenadata/github_mirroring/gpdb/-/jobs/1869443

**Comment by BenderArenadata** [comment]
Failed job Deploy multiarch Dockerimages: https://gitlab.adsw.io/arenadata/github_mirroring/gpdb/-/jobs/1869764

**Comment by BenderArenadata** [comment]
Failed job Deploy multiarch Dockerimages: https://gitlab.adsw.io/arenadata/github_mirroring/gpdb/-/jobs/1869766

**Comment by BenderArenadata** [comment]
Failed job Deploy multiarch Dockerimages: https://gitlab.adsw.io/arenadata/github_mirroring/gpdb/-/jobs/1869795

**Comment by swork12** [comment]
bender build

**Comment by BenderArenadata** [comment]
Allure report https://allure.adsw.io/launch/79509

**Pull request merged**